components/ErrorEventsDashboard.js

```diff
@@ -12,7 +12,8 @@ function ErrorEventsDashboard () {
   const limit = 25
   const url = `${ERROR_EVENTS_URL}?offset=${offset}&limit=${limit}`
   const result = useSWR(url)
-  const { data: events } = result
+  const { data: swrData = {} } = result
```

Comment:
We seem to provide this default value `{}` every time we call `useSWR`. Perhaps we should just use `secureFetch` return an object that mimics the ResponseList from otp-middleware instead. E.g.:

```javascript
if (error) {
  return {
    data: {data: [], total: -1, limit: -1, offset: -1}
    message,
    status: 'error'
  }
}
```

Reply:
It is not so simple, as the `data` field of the useSWR result can be null if SWR has not invoked the fetcher yet. (see https://swr.vercel.app/docs/options)